index.html

```diff
@@ -4,9 +4,46 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./client/assets/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <title>project!</title>
 
     <link href="./styles/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+
+  <body>
+    <header class="Header">
+      <div class="container">
+        <div class="Header-in">
```

Comment:
be consistent in naming your CSS classes, you have classes that start with smaller letters and then you used also capital letters.
```
Header
container
```

Reply:
container is a generic class. and container a css manipulation is not done. but those that start with a capital letter are special classes and include only the section in which they are found. this is a general principle.

Reply:
It would be better to use small letters.